pkgs/by-name/an/anytype/package.nix

```diff
@@ -31,6 +32,12 @@ let
     rev = "822f8ea833a94fb48cd8e304ef8dc557b67a9f7b";
     hash = "sha256-fum8zLRXb8xW8TwNyelIZVZR6XXsdPHSt1WDo+TX4CU=";
   };
+
+  electron-headers = runCommand "electron-headers" { } ''
+    mkdir -p $out
+    tar -C $out --strip-components=1 -xvf ${electron.headers}
+  '';
```

Comment:
We should move this inside of electron tbh

Reply:
I agree

Reply:
#385341